frontend/src/components/App/PluginSettings/PluginSettings.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import { Meta, Story } from '@storybook/react/types-6-0';
+import React from 'react';
+import store from '../../../redux/stores/store';
+import { TestContext } from '../../../test';
+import { PluginSettingsPure, PluginSettingsPureProps } from './PluginSettings';
+
+export default {
+  title: 'Settings/PluginSettings',
+  component: PluginSettingsPure,
+} as Meta;
+
+const Template: Story<PluginSettingsPureProps> = args => (
+  <TestContext store={store}>
+    <PluginSettingsPure {...args} />
+  </TestContext>
+);
+
+/** createDemoData function will create example data objects to act as plugin data
+ *  the function will return an array of demo data objects based on the number specified*/
+function createDemoData(arrSize: number) {
+  // Static list of plugins
+  const pluginArr: any = [];
+  for (let i = 0; i < arrSize; i++) {
+    const newPlugin = {
+      name: `plugin a ${i}`,
+      description: `This is a plugin for this project PLUGIN A${i}`,
+      homepage: `https://plugin-link-${i}`,
+      isEnabled: i % 2 === 0,
+    };
+    pluginArr.push(newPlugin);
+  }
+  return pluginArr;
+}
+
+/** Creation of data arrays ranging from 0 to 50 to demo state of empty, few, many, and large numbers of data objects.
+ * NOTE: The numbers used are up to the users preference.
+ */
+const demoEmpty = createDemoData(0);
+const demoFew = createDemoData(5);
+const demoMany = createDemoData(15);
+const demoMore = createDemoData(50);
+
+/** Use console inspect to track console log messages which will act as labels to
+ *  for each action that is being performed and in which story after pressing the save button */
+export const Empty = Template.bind({});
```

Comment:
I noticed putting the empty(or closed) state first is often confusing to a person browsing storybook. Because it shows nothing when they click on it, and the first state is shown.